### DISCUSSION TITLE
Expander expand and collapse delays

### DISCUSSION THREAD

**Discussion by PontusLindberg** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3335] [Q&A] [answered]
Hi! Is it possible for me to adjust the expand and collapse delays for the Expander? If so, how? 

Kind regards,  Pontus

**Comment by nicolaihenriksen** [comment, marked as answer]
@PontusLindberg Unfortunately I don't think there is an easy way for you to modify these without re-doing (copy+modify) the `Style` for `Expander` and the associated `ToggleButton` yourself.

You can see the delays defined [here](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/29bcb91d6c68203d9515bd659c4d6f16ff35b4a5/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Expander.xaml#L6-L10), and below them you'll find the styles which use them. You effectively need to override these styles for your instance(s) of the `Expander` and provide another delay. This has the down-side that you opt-out of any tweaks/modifications that happens to the MDIX styles in future releases; you'll manually need to resync your own implementation.

**Reply by PontusLindberg** [reply]
Ok, thankyou!

**Comment by JoachimEhbrecht** [comment]
Hello,
I had the same requirement.
Did the access of the durations changes since October 2023, or is still copy-paste needed?

The problem happens on control loading, when the default for the IsExpanded property is set to true.
Maybe there is another workaround?

Best regards, Joachim
